modules/ROOT/pages/websockets/websocket-about-websocket-connector.adoc

```diff
@@ -0,0 +1,60 @@
+= About the Websockets Connector (Design Center)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== Why use Websockets
+
+WebSockets are a key piece of modern integration due to their ability to:
+
+* Seamlessly work on top of the existing HTTP infrastructure.
+* Avoid polling
+* Implement server push patterns
+* Establish bi-directional and full duplex communication between server and client.
+
+== Basic components
+
+There are two basic socket types:
+
+* Inbound: represents a websocket that will open upon receiving an incoming connection request from a client that supports websocket communication.
+* Outbound: represents a websocket that will open a connection to other websockets through the 'connect' operation.
+
+All websockets, both inbound and outbound, will have their own unique socket ID to identify them.
+
+== Basic Operations Overview
+
+Using the WS or WSS protocol, the Websocket connector performs one of the following operations:
```

Comment:
This is using the WS and WSS terms without explaining the concepts. At this point, the user has no idea that websocket URLs use the ws/wss schemes nor their relationship to TLS